Voycar.Api.Web/Features/Members/Endpoints/Put/Personal/Mapper.cs

```diff
@@ -0,0 +1,27 @@
+namespace Voycar.Api.Web.Features.Members.Endpoints.Put.Personal;
+
+using Entities;
+
+
+public class Mapper : ResponseMapper<Request, Member>
```

Comment:
```suggestion
public class Mapper : ResquestMapper<Request, Member>
```
Auch wenn hier beides funktioniert, willst du eigentlich einen `RequestMapper`, wenn du `ToEntity()` benutzt.

Reply:
Ah ja, da war vorhin was mit den commits xD Ja, passe ich an. Danke